v2/concordium/types.proto

```diff
@@ -1306,6 +1369,63 @@ message DelegationEvent {
   }
 }
 
+//Token events originating from account transcations.
+message TokenEvent {
+  // A token transfer between two accounts.
+  message Transfer {
+    TokenAmount amount = 1;
+    AccountAddress from_address = 2;
+    AccountAddress to_address = 3;
+    optional string memo = 4;
+  }
+
+  oneof event {
+    Transfer transfer = 1;
+  }
+}
+
+//Token events originating from governance transactions.
+message TokenGovernanceEvent {
+  message Mint {
+    TokenId token_id = 1;
+    TokenAmount amount = 2;
+  }
+
+  message Burn {
+    Token token_id = 1;
```

Comment:
Similar to mint, we probably need some address from which these are burned from.